### PR TITLE
fix touchscreen bug

### LINKED ISSUE #3167: Save button remains disabled after a node position is changed
**Describe the bug**
Issue with saving position of nodes from version 0.172.0
While editing a workflow, after a node is moved, the Save button remains disabled.
If some other change is made to the workflow, e.g. a node is deactivated, the Save button is enabled. But when saving the workflow, the position of the nodes, which was changed before, is ignored. After refreshing the page, the position of the nodes is different from what was in the editor before.
This issue is not in version 0.171.0

**To Reproduce**
1. Go to 'Workflows / New'
2. Click on 'Add Notes / Flow / IF'
3. Click on 'Save'
4. Move the 'IF' node
5. Disable the 'IF node
6. Click on 'Save'

**Expected behavior**
After step 4: The 'Save' button is enabled
After step 6: Position of the nodes is saved

**Environment (please complete the following information):**
 - Docker: n8nio/n8n
 - Tags: 0.172.0, 0.173.1
 - Database system: SQLite
 - Operation mode: own

**Additional context**

### PR DESCRIPTION
**Pull request by mutdmour**
reverting ts related changes  https://github.com/n8n-io/n8n/commit/027dfb2f0aa8faa0cb109ab41b0c3bd9c1ea8450#diff-3d3c041f8ea21c808cb848b1e406293cfb11c9b09c68b67b81389ea0469db1a9R6

fixes https://github.com/n8n-io/n8n/issues/3167

Community Issue:
https://community.n8n.io/t/172-0-nodes-position-not-saved/13082/6

### CODE REVIEW

File: packages/editor-ui/src/components/mixins/deviceSupportHelpers.ts [hunk 1]
```diff
@@ -3,7 +3,8 @@ import Vue from 'vue';
 export const deviceSupportHelpers = Vue.extend({
 	data() {
 		return {
-			isTouchDevice: 'ontouchstart' in window || navigator.maxTouchPoints,
+			// @ts-ignore msMaxTouchPoints is deprecated but must fix tablet bugs before fixing this.. otherwise breaks touchscreen computers
+			isTouchDevice: 'ontouchstart' in window || navigator.msMaxTouchPoints,
```
Comment (ivov) [inline packages/editor-ui/src/components/mixins/deviceSupportHelpers.ts:7]: Oddly, `msMaxTouchPoints` and `platform` are both deprecated but `msMaxTouchPoints` returns `undefined` and `platform` still returns a string. Wondering if we might have an issue once `platform` starts returning `undefined`.

**Review by ivov — COMMENTED** [review]
For reference I don't have a touchscreen to test the fix.

File: packages/editor-ui/src/components/mixins/deviceSupportHelpers.ts (hunk 1 above)
Comment (mutdmour) [reply packages/editor-ui/src/components/mixins/deviceSupportHelpers.ts:7]: yes would have to handle platform separately as tech debt

**Comment by janober** [comment]
Got released with `n8n@0.175.0`

**Pull request merged**